buildmaster/old_new_porting_map.yml

```diff
@@ -0,0 +1,382 @@
+  new_name: ATLAS_SINGLETOP_8TEV_TBAR-RAP-NORM
+  reference_arxiv: '1702.02859'
+  journal: Eur. Phys. J. C 77 (2017) 531
+ATLAS_SINGLETOP_TCH_DIFF_8TEV_T_RAP_NORM:
+  new_name: ATLAS_SINGLETOP_8TEV_T-RAP-NORM
+  reference_arxiv: '1702.02859'
+  journal: Eur. Phys. J. C 77 (2017) 531
+ATLAS_SINGLETOP_TCH_R_13TEV:
+  new_name: ATLAS_SINGLETOP_13TEV_TCHAN-XSEC
+  reference_arxiv: '1609.03920'
+  journal: JHEP 04 (2017) 086
+ATLAS_SINGLETOP_TCH_R_7TEV:
+  new_name: ATLAS_SINGLETOP_7TEV_TCHAN-XSEC
+  reference_arxiv: '1406.7844'
+  journal: Phys. Rev. D 90 (2014) 112006
+ATLAS_SINGLETOP_TCH_R_8TEV:
+  new_name: ATLAS_SINGLETOP_8TEV_TCHAN-XSEC
+
+# FTDY (non dw/ missing)
+DYE605_dw_ite:
+  new_name: DYE605_Z0_38P8GEV_DW_PXSEC
+  reference_arxiv: ''
+  journal: Phys. Rev. D43 (1991) 2815
+DYE886R_dw_ite:
+  new_name: DYE866_Z0_800_RATIO_DW_PDXSECRATIO
+  reference_arxiv: hep-ex/0103030
+  reference_hepdata: 10.17182/hepdata.22184/t4
+  journal: Phys. Rev. D64 (2001) 052002
+DYE886P:
+  new_name: DYE866_Z0_800_PXSEC
+  reference_arxiv: hep-ex/0302019
+DYE906R_dw_ite:
+  new_name: DYE906_Z0_120GEV_DW_PDXSECRATIO
+  reference_arxiv: '2103.04024'
+  journal: Nature 590 (2021), no. 7847 561
+CHORUSNB:
+  new_name: CHORUS_CC_AVG98GEV_NBPB
+  reference_arxiv: ''
+  reference_hepdata: 10.17182/hepdata.6187.v1
+CHORUSNBPb_dw_ite:
+  new_name: CHORUS_CC_AVG98GEV_DW_NBPB
+  reference_arxiv: ''
+  journal: Phys. Lett. B632 (2006) 65
+CHORUSNU:
+  new_name: CHORUS_CC_AVG98GEV_NUPB
+  reference_arxiv: ''
+  reference_hepdata: 10.17182/hepdata.6187.v1
+CHORUSNUPb_dw_ite:
+  new_name: CHORUS_CC_AVG98GEV_DW_NUPB
+  reference_arxiv: ''
+  reference_hepdata: 10.17182/hepdata.6187.v1
+  journal: Phys. Lett. B632 (2006) 65
+EMCF2C_dw_ite:
+  new_name: EMC_NC_250GEV_EM-F2C
+  reference_arxiv: ''
+
+# NMC
+# TODO: check energy
+NMC:
+  new_name: NMC_NC_AVG185GEV_EMP
+  reference_arxiv: hep-ph/9610231
+  reference_hepdata: 10.17182/hepdata.32750.v1
+  journal: Nucl. Phys. B483 (1997) 3
+NMCPD:
+  new_name: NMC_NC_AVG185GEV_EMD-F2
+  reference_arxiv: hep-ex/9611022
+  reference_hepdata: 10.17182/hepdata.32750.v1
+NMCPD_dw_ite:
+  new_name: NMC_NC_AVG185GEV_DW_EMD-F2
+  reference_arxiv: hep-ex/9611022
+  reference_hepdata: 10.17182/hepdata.32750.v1
+  journal: Nucl. Phys. B487 (1997) 3
+NTVNBDMN:
+  new_name: NUTEV_CC_AVG176GEV NBFE
+  reference_arxiv: hep-ex/0102049
+  reference_hepdata: 10.17182/hepdata.42034.v1
+NTVNBDMNFe_dw_ite:
+  new_name: NUTEV_CC_AVG176GEV_DW_NBFE
+  reference_arxiv: hep-ex/0102049
+  reference_hepdata: 10.17182/hepdata.42034.v1
+  journal: Phys. Rev. D64 (2001) 112006
+NTVNUDMN:
+  new_name: NUTEV_CC_AVG176GEV_NUFE
+  reference_arxiv: hep-ex/0102049
+  reference_hepdata: 10.17182/hepdata.42034.v1
+NTVNUDMNFe_dw_ite:
+  new_name: NUTEV_CC_AVG176GEV_DW_NUFE
+  reference_arxiv: hep-ex/0102049
+  reference_hepdata: 10.17182/hepdata.42034.v1
+  journal: Phys. Rev. D64 (2001) 112006
+
+# SLAC
+# TODO: check energy
+SLACD:
+  new_name: SLAC_NC_AVGXXGEV_DF2
```

Comment:
I guess we want to separate target `D` and obs `F2`, no?

Reply:
RE this, let's talk about it during the meeting today. This one (and everything with nuclear corrections really) need some choices to be made...

Reply:
nuclear corrections are a complete chapter on their own, but with the [discussion from above](https://github.com/NNPDF/nnpdf/pull/1931#discussion_r1481482698) this then should be something `F2-D`

Reply:
For now I have separated by target (so, `_D_F2` and `_P_F2` have independent metadata  but `_D_F2-DW` and `_D_F2` share it.

But we can consolidate it (by doing `_F2-D` instead)